splitio/storage/pluggable.py

```diff
@@ -10,12 +10,16 @@
 class PluggableSplitStorage(SplitStorage):
     """InMemory implementation of a split storage."""
 
-    def __init__(self, pluggable_adapter, prefix):
+    def __init__(self, pluggable_adapter, prefix=None):
         """Constructor."""
         self._pluggable_adapter = pluggable_adapter
-        self._prefix = prefix + ".split."
-        self._traffic_type_prefix = prefix + ".trafficType."
-        self._split_till_prefix = prefix + ".splits.till"
+        self._prefix = "split."
```

Comment:
I now remember from JS, that the default prefix is `"SPLITIO."` (in uppercase), as you can check here https://github.com/splitio/python-client/blob/master/splitio/storage/redis.py#L21

Reply:
fixed